baker/algolia/indexToAlgolia.ts

```diff
@@ -113,6 +126,46 @@ const getPagesRecords = async () => {
         }
     }
 
+    for (const gdoc of gdocs) {
+        // Only rendering the blocks - not the page nav, title, byline, etc
+        const renderedPostContent = ReactDOMServer.renderToString(
```

Comment:
```suggestion
        const renderedPostContent = ReactDOMServer.renderToStaticMarkup(
```

Better to use `renderToStaticMarkup` here. See https://stackoverflow.com/a/67778035 for the difference between the two (I'm also always confusing the two!).